code/src/App.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react'
 import data from './data.json'
+import Album from "./components/Album"
```

Comment:
There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.

Reply:
Done.